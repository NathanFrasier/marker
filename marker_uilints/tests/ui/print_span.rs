macro_rules! magic_macro {
    () => {
        "*magic penguin noises*"
    };
}
use magic_macro;

fn main() {
    let _span_normal = 178;

    let _span_macro = magic_macro!();
}
