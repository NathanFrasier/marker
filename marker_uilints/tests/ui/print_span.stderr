warning: print span
 --> $DIR/print_span.rs:9:5
  |
9 |     let _span_macro = magic_macro!();
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: Debug: Span {
              source: Macro(
                  SpanSrcId(..),
              ),
              start: 0,
              end: 14,
          }
  = note: Snippet: magic_macro!()
  = note: `#[warn(marker::test_lint)]` on by default

warning: print span
  --> $DIR/print_span.rs:11:5
   |
11 |     let _span_normal = 178;
   |     ^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: Debug: Span {
               source: File(
                   "$DIR/print_span.rs",
               ),
               start: 177,
               end: 180,
           }
   = note: Snippet: 178

warning: 2 warnings emitted

